node/manifest.ts

```diff
@@ -18,6 +18,7 @@ interface Route {
   name?: string
   pattern: string
   generator?: string
+  on_error?: string
```

Comment:
this is the property name that'll be used in nf-server / stargate - let's get this right on the first attempt. I chose  `on_error`, since that makes the relation to the user-facing feature obvious. But i've also seen `failure mode` being used - what term should we use in code? @eduardoboucas leaving that decision to you.

Reply:
I think `onError` is a more intuitive user-facing name, but I do like "failure mode" when referring to the feature itself (otherwise, do we say that a function has an "on error"?). But I see your point about having a consistent name across the board, in which case I think I'd go with "on error".

What do you think?

Reply:
I think it's fine to talk about "failure modes" as an attribute of a deployment, but when talking about *this specific feature*, *this specific config setting*, I like calling it "`onError` setting".
"failure modes" is already an overloaded term - it's a functionality that `onError` enables, but there could be other config settings in the future that contribute to a function's failure mode.
In our domain language, I think we should say that "you can use the `onError` setting to tune your edge function's failure modes". Following that, we should use `on_error` as the internal config value.